org-formation/600-access/_tasks.yaml

```diff
@@ -186,53 +186,15 @@ FPTMinimumAccess:
             "Effect": "Allow",
             "Action": ["bcm-data-exports:*"],
             "Resource": [
-                  "Fn::Sub": "arn:aws:bcm-data-exports:${AWS::Region}:${AWS::AccountId}:export/*",
-                  "Fn::Sub": "arn:aws:bcm-data-exports:${AWS::Region}:${AWS::AccountId}:table/COST_AND_USAGE_REPORT"
+                  "arn:aws:bcm-data-exports:*:*:export/*",
+                  "arn:aws:bcm-data-exports:*:*:table/COST_AND_USAGE_REPORT"
                   ]
           },
           {
             "Sid": "CurDataAccess",
             "Effect": "Allow",
             "Action": ["cur:PutReportDefinition"],
             "Resource": "*"
-          },
```

Comment:
these are resource policies.  they have been moved to the bucket.  it was manually added to the bucket policy because this bucket was not created with cloudformation